### DISCUSSION TITLE
Can't run FireFox with Sandboxie Classic

### DISCUSSION THREAD

**Discussion by DocBadwrench** [sandboxie-plus/Sandboxie#1351] [Q&A Feedback]
Or the better version. I tried both. Every time I attempt to run that browser in sandboxed mode, I get a Sandboxie Start dialog box that reads:

> "[PATH]\firefox.exe"
> System Error Code:
> The application has failed to start because its side-by-side configuration is incorrect. Please see the application event log or use the command-line sxstrace.exe tool for more detail. (14001)

I have reinstalled both Firefox and Sandboxie respectively, but no dice. I'm happy to provide further info; just let me know what.

**Comment by isaak654** [comment]
I can't reproduce it, did you try to create a new sandbox with default settings?
